rasa/core/policies/ted_policy.py

```diff
@@ -379,10 +386,10 @@ def model_class() -> Type["TED"]:
     def _metadata_filename(cls) -> Optional[Text]:
         return "ted_policy"
 
-    def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
-        new_config = rasa.utils.train_utils.check_core_deprecated_options(kwargs)
+    def _load_params(self, config: Dict[Text, Any]) -> None:
+        new_config = rasa.utils.train_utils.check_core_deprecated_options(config)
         self.config = rasa.utils.train_utils.override_defaults(
-            self.defaults, new_config
+            self.default_config, new_config
```

Comment:
Again, isn't this covered in `GraphNode`?

Reply:
done